api/src/modules/impact/impact.service.ts

```diff
@@ -465,4 +480,27 @@ export class ImpactService extends BaseImpactService {
       isProjected: false,
     };
   }
+
+  private getLevelOfImpactTableRows(
+    impactTableRows: ImpactTableRows[],
+    level: number,
+  ): ImpactTableRows[] {
+    if (level === 1) {
+      return impactTableRows;
+    }
+    if (level < 1) {
```

Comment:
I think this is not necessary, because:
1. first level should be 0, which is a valid value
2. there should be a default behaviour to default to